extensions/blocks/mailchimp/mailchimp-groups.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { CheckboxControl } from '@wordpress/components';
+import { Fragment, Component } from '@wordpress/element';
+
+class MailchimpGroups extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			groups: [],
+		};
+	}
```

Comment:
We've been avoiding using `constructor/super` in other blocks, this works as well:

https://github.com/Automattic/jetpack/blob/f25df0ee2d9da0159bd093fe849a10454a0caacc/extensions/blocks/business-hours/edit.js#L32-L37

Reply:
Sure, I'll make the change. What's the reasoning?

Reply:
The variable name was incorrect too 😬. Fixed in https://github.com/Automattic/jetpack/pull/14050/commits/b0c909297619fe3f14f592c1801e8d639e9fd031